themes/sensei-course-theme/single-quiz.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Quiz template.
+ *
+ * @author      Automattic
+ * @package     Sensei
+ * @category    Templates
+ * @version     4.0.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+?>
+<!-- wp:sensei-lms/container {"className":"sensei-course-theme__quiz"} -->
+<div class="wp-block-sensei-lms-container sensei-course-theme__header sensei-course-theme__frame">
+	<!-- wp:sensei-lms/quiz-back-to-lesson /-->
+	<!-- wp:post-title /-->
+	<!-- wp:post-content /-->
```

Comment:
I have a feeling that you explained the purpose of this in today's meeting and I didn't get it. haha
So if yes, could you explain again why we have the title and post content blocks here for the quiz?

Reply:
We'll probably have to make our own copy of these to support all Wordpress versions, but they are in there as blocks (instead of `<?php the_content(); ?>` ) to move closer to being a block-only template.

But in this PR this file is just a placeholder and `post-content` block doesn't really work with how we render the quiz via templates. 

In the next PR, https://github.com/Automattic/sensei/pull/4495, I added a `sensei-lms/the-content` block that has all the PHP bits (like the check if user can access the lesson), and also renders the quiz using the current templates and hooks for the questions loop.